include/uri/up_uri_factory.h

```diff
@@ -35,12 +35,12 @@
 namespace uri_factory {
 class up_uri_factory {
  public:
-  static std::string buildUProtocolUri(const uri_datamodel::up_uri& upUri) {
+  static std::string buildUProtocolUri(const uri_datamodel::UUri& upUri) {
     if (upUri.isEmpty()) {
-      return uri_datamodel::up_uri::SCHEME;
+      return std::string("up:");
```

Comment:
Instead of repeating "up", we could have the default shema

Reply:
i am going to revert this as it should not be part of the PR